Index: git03/src/main/java/com/itheima/test/Demo.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- git03/src/main/java/com/itheima/test/Demo.java	(revision 9b285ea30bbc52073671c842a86bcce99649f337)
+++ git03/src/main/java/com/itheima/test/Demo.java	(revision ced891949a285348ddbaed2713eef1b7c8a613eb)
@@ -1,5 +1,7 @@
 package com.itheima.test;
 
+import com.itheima.pojo.Dept;
+
 /**
  * @program: git03
  * @author: wenlong
@@ -13,5 +15,11 @@
         System.out.println(did=9);
         System.out.println(did=10);
         System.out.println(21111);
+        Dept dept=new Dept();
+        dept.setDid(1);
+        dept.setDname("张三");
+        dept.setDtel("31457-7894");
+        String s = dept.toString();
+        System.out.println(s);
     }
 }
